test/integration/models/networking/test_networking.py

```diff
@@ -73,6 +77,47 @@ def test_get_networking_rules(test_linode_client, test_firewall):
     assert "outbound_policy" in str(rules)
 
 
+@pytest.fixture
```

Comment:
I only added some fairly minimal integration testing here since a lot of it would depend on the new interface resources being supported, which will be done in a future ticket.